neural_compressor/adaptor/onnxrt.py

```diff
@@ -181,6 +181,7 @@ def smooth_quant(
             "alpha_step": 0.1,
             "shared_criterion": "mean",
             "do_blockwise": False,
+            "shift_bias": False,
```

Comment:
how about using **kwargs, otherwise, each time we need to add the keywords for different backends

Reply:
auto_alpha_args is already added to kwargs in /algorithm/smooth_quant.py.
Just adding the arguments and related docstrings for onnx and tensorflow.